docs/swagger-ui-http4s/src/main/scala/sttp/tapir/swagger/http4s/SwaggerHttp4s.scala

```diff
@@ -24,11 +24,11 @@ import scala.concurrent.ExecutionContext
   */
 class SwaggerHttp4s(
     yaml: String,
-    contextPath: String = "docs",
+    contextPath: List[String] = List("docs"),
```

Comment:
Kept for compatibility with the previous version when the context path is not defined.